examples/Cargo.toml

```diff
@@ -35,7 +35,7 @@ gfx-hal = { path = "../src/hal", version = "0.2" }
 gfx-backend-empty = { path = "../src/backend/empty", version = "0.2" }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-winit = "0.19"
+winit = "0.20.0-alpha2"
```

Comment:
do I understand correctly that we are waiting for "0.20.0" to come out before merging this?

Reply:
Yes.

Alternatively it might be nice to get this merged after the release of gfx Hal 0.3